src/emcpy/plots/create_map.py

```diff
@@ -0,0 +1,294 @@
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.pyplot as plt
+from matplotlib import rcParams
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+from map_tools import Domain, MapProjection
+
+
+__all__ = ['CreateMap']
+
+
+class CreateMap:
+    """
+    Creates a map axes on a figure to plot data.
+    """
+
+    def __init__(self, fig=plt.figure(figsize=(12,8)),
+                 domain=Domain('global'),
+                 projection=MapProjection('plcarr')):
+        """
+        CreateMap constructor
+
+        Args:
+            fig : (matplotlib.figure) Figure object to plot on
+            domain : (object) domain from emcpy Domain()
```

Comment:
these should probably be emcpy.plots.map_tools.Domain() or however one would actually import them from scratch.